### ISSUE TITLE
Analyze segment sizes per user agent

### ISSUE THREAD

**Issue by wthorp** [storj/edge#77]
See if we can get uploaded segment size data for Gateway-MT specifically, and if we can correlate this with user agent information somehow.  If not, figure out how we can gather this information and implement it.

If we have the necessary information, create a draft blog post on how to configure these clients to use larger segment sizes.  If there is development work / metrics that need to be added before we can understand this, spin this second part out into a new ticket.
